sqlglot/dialects/doris.py

```diff
@@ -49,6 +49,8 @@ class Generator(MySQL.Generator):
             exp.ArrayUniqueAgg: rename_func("COLLECT_SET"),
             exp.CurrentTimestamp: lambda self, _: self.func("NOW"),
             exp.DateTrunc: lambda self, e: self.func("DATE_TRUNC", e.this, unit_to_str(e)),
+            exp.GroupConcat: lambda self,
+                                    e: f"""GROUP_CONCAT({self.sql(e, "this")} , {self.sql(e, "separator") or "','"})""",
```

Comment:
exp.func("GROUP_CONCAT", e.this, e.args.get("seprator") or ",")

Reply:
yes , but need to add ''

Reply:
exp.Literal.string(",")